ale_linters/ruby/reek.vim

```diff
@@ -0,0 +1,48 @@
+" Author: Eddie Lebow https://github.com/elebow
+" Description: Reek, a code smell detector for Ruby files
+
+function! ale_linters#ruby#reek#Handle(buffer, lines) abort
+    if len(a:lines) == 0
+        return []
+    endif
+
+    let l:errors = json_decode(a:lines[0])
+
+    let l:output = []
+
+    for l:error in l:errors
+        for l:location in l:error.lines
+            call add(l:output, {
+            \    'bufnr': a:buffer,
```

Comment:
You don't have to set `bufnr` if you don't want to. That key will be set automatically by ALE later on.